CONTRIBUTING.md

```diff
@@ -239,6 +239,16 @@ style of the existing OpenLayers 3 code, which includes:
 
  * Do not use assignments inside expressions.
 
+ * Avoid the use of `goog.array.clone` with arrays (use slice instead).
+
+ * Use `array.length = 0` instead of `goog.array.clear`.
+
+ * Use bracket notation instead of `goog.object.set` and `goog.object.get` (with
+   two arguments).
+
+ * Use `opt_` prefix for optional parameters.
```

Comment:
This is enforced by the linter, but it's also ok to have it here I guess.

I'd use "for optional function parameters" rather than "for optional parameters".